### PR TITLE
[PP-24] Show user roles in room members list

### PR DESCRIPTION
**Pull request by andreikaptsiuh**
Task: https://andrei-lysiuk.atlassian.net/browse/PP-24
Description: Show user roles in room members list

### CODE REVIEW

File: src/components/CommonArea/CardNest/CardNest.tsx [hunk 1]
```diff
@@ -6,26 +6,26 @@ interface ICardNestProps {
...
+  userRole,
 }) => (
   <section
     className={classNames(
       styles.cardNest,
       className,
       {
         [styles.showCard]: isCardOpened,
-        [styles.spectator]: userType === 'spectator',
+        [styles.spectator]: userRole === 'spectator',
```
Comment (mrakofff) [inline src/components/CommonArea/CardNest/CardNest.tsx:22]: the same condition will be for `roll: null`

File: src/components/User/User.tsx [hunk 2]
```diff
@@ -16,7 +17,14 @@ export const User: FC<IUserProps> = ({ user }) => {
 
   return (
     <li className={styles.item}>
-      <div className={styles.user}>
+      <div className={classNames(
+        styles.user,
+        {
+          [styles.user__admin]: user.role === 'admin',
+          [styles.user__spectator]: user.role === 'spectator',
```
Comment (mrakofff) [inline src/components/User/User.tsx]: roll is optional(could be null)

**Review by mrakofff — APPROVED** [review]
if the same will work with optional role it could be merged

**Review by MakFill — APPROVED** [review]

File: src/components/CommonArea/CardNest/CardNest.tsx (hunk 1 above)
Comment (andreikaptsiuh) [reply src/components/CommonArea/CardNest/CardNest.tsx:22]: fix

File: src/components/User/User.tsx (hunk 2 above)
Comment (andreikaptsiuh) [reply src/components/User/User.tsx]: fix

**Pull request merged**